### PR TITLE
Feature/basic tune playing

### PR DESCRIPTION
**Pull request by JR-Mitchell**
- Added note struct for storing frequency & duration data
- Added tune struct for playing notes in order
- Implement and use a placeholder title theme

### CODE REVIEW

File: src/music/title-theme.zig [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Placeholder implementation - not complete
+const tone = @import("tone.zig");
```
Comment (jwiggins) [inline src/music/title-theme.zig:2]: You forgot to add this file to git

**Review by jwiggins — DISMISSED** [review]

**Review by jwiggins — COMMENTED** [review]
Other than this missing file this looks good

File: src/music/title-theme.zig (hunk 1 above)
Comment (JR-Mitchell) [reply src/music/title-theme.zig:2]: Ah bother

Comment (JR-Mitchell) [reply src/music/title-theme.zig:2]: Will do in the morning

Comment (JR-Mitchell) [reply src/music/title-theme.zig:2]: Just realised - this file already exists at main.

Comment (JR-Mitchell) [reply src/music/title-theme.zig:2]: You happy for me to merge?

**Comment by JR-Mitchell** [comment]
Gonna merge this now, as the missing file was a red herring (already exists on main)

**Comment by jwiggins** [comment]
> already exists on main

Shit. Sorry about that. I thought this was the first bits.

**Comment by jwiggins** [comment]
LGTM

**Pull request merged**